hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/fairness/ProportionRouterRpcFairnessPolicyController.java

```diff
@@ -93,7 +93,8 @@ public boolean acquirePermit(String nsId) {
   public void releasePermit(String nsId) {
     if (contains(nsId)) {
       super.releasePermit(nsId);
+    } else {
+      super.releasePermit(DEFAULT_NS);
     }
```

Comment:
Personal opinion, whether it can be written like this.
```java
if (contains(nsId)) {
   super.releasePermit(nsId);
   return;
}
super.releasePermit(DEFAULT_NS);
```

Reply:
yes, thank you for your reply,the effect is the same. : )